go/cmd/vtcombo/plugin_dbddl.go

```diff
@@ -29,7 +30,9 @@ var globalCreateDb func(ctx context.Context, ks *vttestpb.Keyspace) error
 var globalDropDb func(ctx context.Context, ksName string) error
 
 // DBDDL doesn't need to store any state - we use the global variables above instead
-type DBDDL struct{}
+type DBDDL struct {
+	createDbLock *sync.Mutex
```

Comment:
Small nit, Vitess uses `mu` as the member name for mutexes in structs.

I also think you can use a value here instead of a pointer:

```suggestion
	mu sync.Mutex
```